### DISCUSSION TITLE
Cant Update Docker Installation

### DISCUSSION THREAD

**Discussion by shaunm001** [requarks/wiki#5867] [Help / Questions] [answered]
Hey everyone, we're trying to use the Admin console to update our Docker installation:

![image](https://user-images.githubusercontent.com/27734187/202760475-e13576a1-4f5d-4467-8ba5-410878589643.png)

But when we click the "Perform Upgrade" button, it goes through the motions like it's doing the upgrade, then after it finishes we're brought back to the Admin console and it still shows the same "Current" and "Latest" versions as if we never did the upgrade. 

Worse, clicking "Perform Upgrade" blows away our custom SSL config. 

Anyone else run into similar issues? Any suggestions on how best to install upgrades?

**Comment by NGPixel** [comment, marked as answer]
https://docs.requarks.io/en/troubleshooting#unable-to-upgrade-from-the-administration-area

**Reply by shaunm001** [reply]
Thanks! This did allow me to upgrade using the admin area, but it still blows away my custom SSL config. I will go through and configure it again, but just wanted to share.
